client/task-list/index.js

```diff
@@ -306,7 +315,14 @@ class TaskDashboard extends Component {
 			}
 
 			if ( ! task.onClick ) {
-				task.onClick = () => updateQueryString( { task: task.key } );
+				task.onClick = ( e ) => {
+					if ( e.target.nodeName === 'A' ) {
```

Comment:
Note that this is never actually exercised since every task in `tasks.js` has an `onClick` handler. To test this, add the following to the end of the tasks array in `tasks.js`:

```
{
			key: 'test',
			title: 'Test task',
			container: <Appearance/>,
			completed: false,
			visible: true,
			additionalInfo: 'test with a <a href="https://google.com" target="_blank">link</a>',
			time: '2 minutes',
		},
```